### PR TITLE
Frontend changes

### PR DESCRIPTION
**Pull request by kcwoolverton**
I don't know what the protocol here is--this is my frontend branch, which was branched of the main frontend branch. I think it's good, but I don't know if I should merge with the main one first, then go to dev, or try to merge directly to dev. If I did it wrong, please let me know.

### CODE REVIEW

File: app/views/assignments/index.html.erb [hunk 1]
```diff
@@ -1,33 +1,43 @@
...
-                  <td class="col-lg-2" align="center"> 
-                        <a ><%= link_to 'Show', user_course_assignment_path(@user, @course, assignment) %></a>
-                        <a ><%= link_to 'Edit', edit_user_course_assignment_path(@user, @course, assignment) %></a>
-                        <a ><%= link_to 'Destroy', user_course_assignment_path(@user, @course, assignment),
-                          method: :delete,
-                          data: { confirm: 'Are you sure?' } %> </a>
-                  </td>
+        <table class="table table-hover table-responsive">
+          <thead >
+            <tr class= "row"; style="background-color:rgb(78,55,100,.5);">
```
Comment (mcshen) [inline app/views/assignments/index.html.erb]: It is better to put style in a css class. This is actually a remnant of some old code I thought I fixed. oops. Add "table-header" to the class.

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
+          <br>
+              <a class = "btn" <%= link_to "Edit Course", edit_user_course_path(@user, @course) %></a>
+              <br>
+              <button class="btn" onclick="toggleHide()">Order Students</button>
+              <div id="toggleable"; style = "display:none">
+                <h4>Order Students</h4>
+                <%= form_tag user_course_path(@user, @course) do %>
+                  <%= text_area :flashcard_order, :edit_flashcard_order %>
+                  <%= submit_tag "Order Students" %>
+                <% end %>
```
Comment (mcshen) [inline app/views/courses/show.html.erb:42]: nit: I think lines 34-42 are indented too much, and lines 46, 47

File: app/views/courses/student_index.html.erb [hunk 3]
```diff
@@ -41,27 +38,31 @@
...
   </body>
 </center>
-</div>     
+</div>
 </br>
 <center>
   <h4>Import Students</h4>
   <div class = "drop-area">
-    <%= form_tag import_user_course_students_path(@user, @course), 
+    <%= form_tag import_user_course_students_path(@user, @course),
```
Comment (mcshen) [inline app/views/courses/student_index.html.erb:65]: Thanks for getting rid of all the extra spaces.

**Review by mcshen — COMMENTED** [review]
Looks good! I think it'll be easier to keep track of changes if we merge this with the main frontend branch and then merge that with dev.

**Review by Omicron3141 — APPROVED** [review]

**Pull request merged**